src/components/search/index.tsx

```diff
@@ -209,6 +207,7 @@ const Search: FunctionComponent<SearchProps> = ({
 
           <div className="dark:bg-gray-900 bg-gray-50  md:-mt-5">
             <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto dark:bg-gray-900">
+              <ScrollTo scrollOn="page" />
               <Hits />
```

Comment:
moved this here so that paginating on curated page won't force scrolls below curated content